### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by funkyLover**
installation for swift

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ To import pod you should add string:
 - For `Swift` projects:
 
 
-        import CoreActionSheetPicker
+        import ActionSheetPicker_3_0
```
Comment (skywinder) [inline README.md]: Are you sure in that?
Seems you have to write CoreActionSheetPicker for import. Did you try it?

Comment (funkyLover) [inline README.md]: @skywinder actually, I get an error like `no such module named CoreActionSheetPicker` when i `import CoreActionSheetPicker`.
`import ActionSheetPicker_3_0` work fine with Xcode7.3 and swift2.2 for me...

Comment (funkyLover) [inline README.md]: there is so weird..I look for the issue about `swift import`,,seem `import CoreActionSheetPicker` still work for other, however `import ActionSheetPicker_3_0` just work for me

Comment (skywinder) [inline README.md]: Ok, it's wired. I believe We have to leave 2 strings and make some clarification about this behaviour. I believe `ActionSheetPicker_3_0` works for OBjC projects and `CoreActionSheetPicker` for swift project.

**Pull request closed without merging**